spec.bs

```diff
@@ -419,7 +432,7 @@ A [=component=] has an associated <dfn for=component>group name list</dfn>, a [=
     1. Set |hash| to |applyResult|["{{URLPatternInit/hash}}"].
   1. Else:
     1. Let |baseURL| be null.
-    1. If |baseURLString| was given, then:
+    1. If |baseURLString| was given and not null, then:
```

Comment:
Probably better is to change the declaration to "an optional string-or-null |baseURLString| (default null)" and then change this to "is not null".

Reply:
While looking at this I realized this baseURLString is not the same as the constructor baseURL.  I will just revert this change as I think its better just being strictly optional.